File: Java/Array-2/src/Has22.java
public class Has22 {
    public static void main(String[] args) {
        /*
         * Given an array of ints, return true if the array contains a 2 next to a 2 somewhere.
         *
         * has22([1, 2, 2]) → true
         * has22([1, 2, 1, 2]) → false
         * has22([2, 1, 2]) → false
         */
        System.out.println(has22(new int[]{1, 2, 2}));
        System.out.println(has22(new int[]{1, 2, 1, 2}));
        System.out.println(has22(new int[]{2, 1, 2}));
    }

    public static boolean has22(int[] nums) {
        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] == 2 && nums[i] == nums[i + 1]) return true;
        }
        return false;
    }
}

File: Python/List-2/has22.py
def has22(nums):
    """
    Given an array of ints, return True if the array contains a 2 next to a 2 somewhere.

    has22([1, 2, 2]) → True
    has22([1, 2, 1, 2]) → False
    has22([2, 1, 2]) → False
    """
    for i in range(len(nums) - 1):
        if nums[i] == 2 and nums[i + 1] == nums[i]:
            return True
    return False


print(has22([1, 2, 2]))
print(has22([1, 2, 1, 2]))
print(has22([2, 1, 2]))
